client/finality-grandpa/src/communication/mod.rs

```diff
@@ -391,6 +411,24 @@ impl<B: BlockT, N: Network<B>> NetworkBridge<B, N> {
 	}
 }
 
+impl<B: BlockT, N: Network<B>> Future03 for NetworkBridge<B, N>
+where
+	NumberFor<B>: Unpin,
```

Comment:
```suggestion
```
You probably have to add `impl<B, N> Unpin for NetworkBridge<B, N> {}` somewhere instead.

Reply:
Thank you, thank you and thank you @tomaka :heart: